pkg/controller/config/config_controller.go

```diff
@@ -157,51 +168,70 @@ func (r *ReconcileConfig) Reconcile(req reconcile.Request) (reconcile.Result, er
 
 	log.Info("reconciling config change")
 
-	res := &op.Config{}
-	err := r.client.Get(context.TODO(), types.NamespacedName{Name: req.Name}, res)
-
-	// ignore all resources except the `resourceWatched`
-	if req.Name != resourceWatched {
-		log.Info("ignoring incorrect object")
-
-		// handle resources that are not interesting as error
-		if !errors.IsNotFound(err) {
-			r.markInvalidResource(res)
-		}
-		return reconcile.Result{}, nil
-	}
+	config := &op.Config{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: req.Name}, config)
 
 	// handle deletion of resource
 	if errors.IsNotFound(err) {
 		// User deleted the cluster resource so delete the pipeine resources
 		log.Info("resource has been deleted")
-		return r.reconcileDeletion(req, res)
+		return r.reconcileDeletion(req, config)
 	}
 
 	// Error reading the object - requeue the request.
 	if err != nil {
 		log.Error(err, "requeueing event since there was an error reading object")
-		return reconcile.Result{}, err
+		return reconcile.Result{Requeue: true}, err
```

Comment:
we don't have to set the `Requeue`, since `err` should trigger a reconcile IIUC.

Reply:
It's subjective to error, the comment was saying `// Error reading the object - requeue the request.`

Reply:
Probably because https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#reconcile-loop  says so ...; hence you don't have to explicitly set Requeue.

Reply:
ohh yes forget about `err`